closeio/contrib/testing_stub.py

```diff
@@ -246,16 +246,16 @@ def delete_lead(self, lead_id):
         del leads[lead_id]
 
     @parse_response
-    def create_activity_email(self, lead_id, **kwargs):
+    def create_activity_email(self, **kwargs):
```

Comment:
deprecate signature
`warning.warn('The argument "lead_id" has be deprecated in favour of ...", DeprecationWarning)`